core/envelope-sim/src/main/kotlin/fr/sncf/osrd/envelope_sim/etcs/ETCSBrakingCurves.kt

```diff
@@ -47,57 +47,61 @@ fun addBrakingCurvesAtEOAs(
 ): Envelope {
     val sortedEndsOfAuthority = endsOfAuthority.sortedBy { it.offsetEOA }
     var beginPos = envelope.beginPos
-    val builder = OverlayEnvelopeBuilder.forward(envelope)
+    val maxSpeedEnvelope = envelope.maxSpeed
+    var envelopeWithEoaBrakingCurves = envelope
```

Comment:
To distinguish EoA, SvL and stop (min of EoA and SvL):
```suggestion
    var envelopeWithStopBrakingCurves = envelope
```

Reply:
This has disappeared, but it would have been a good point.